src/app/hiv-care-lib/dqa-reports/chart-abstraction-patientlist/chart-abstraction-patientlist.component.ts

```diff
@@ -25,32 +26,29 @@ export class ChartAbstractionPatientlistComponent implements OnInit {
     private _location: Location, private route: ActivatedRoute, ) { }
 
   ngOnInit() {
-    let requestParams: any;
     this.addExtraColumns();
     this.route
       .queryParams
       .subscribe((params) => {
         if (params) {
           this.params = params;
-          requestParams = {
-            locations: this.params.locationUuids,
-            limit: 300,
-            offset: 0
-          };
-
-          this.getPatientList(requestParams);
+          this.getPatientList(this.params);
         }
       }, (error) => {
         console.error('Error', error);
       });
   }
-  private getPatientList(params: any) {
+  private getPatientList(params: any, editAvailable?) {
+    this.isLoading = true;
     this.dqaResource.getDqaChartAbstractionReport(params)
       .subscribe(
         (data) => {
```

Comment:
Perhaps it would be useful to check whether the `data` param in the subscribe call is empty or not and if it is, communicate that to the user by way of error handling or some other approach.